app/javascript/react/screens/App/Overview/Overview.js

```diff
@@ -176,7 +176,7 @@ class Overview extends React.Component {
 
     createTransformationPlanRequestAction(url).then(() => {
       retryMigrationAction(planId);
-      setMigrationsFilterAction('Migration Plans in Progress');
+      setMigrationsFilterAction(__('Migration Plans in Progress'));
```

Comment:
can we hold this until #367 is done? we are already using `%sprintf` for those i believe...

Reply:
just the breadcrumbs for now would be great...